code/src/components/RadioButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const RadioButton = ({ valuedTrait, setValuedTrait, radioSelection }) => {
+  return (
+    <label className="radio-label" htmlFor={radioSelection}>
```

Comment:
Adding tabindex = "0" here after <label or in your ValuedTrait may help to make the tab work when using the Screen Reader. Right now I see that Tab key just work on the first button.